test/unit/server/middleware.server.test.ts

```diff
@@ -94,24 +94,31 @@ describe('Express Middleware', function () {
             })
     })
 
-    it('resets context between requests', function() {
+    it('does not leak context between requests', function() {
         const app = express()
 
         app.use(client.requestHandler)
 
-        app.get('/', function(_req, res) {
-            res.send('Hello World!')
-        })
+        app.get("/:reqId", (req, res) => {
+            const initialContext = client.__store.getStore().context;
```

Comment:
To get proper types, you can use the TestClient class which exposes a `getContext()` method. You can see an example usage in `client.test.ts`.

Reply:
Hm, the TestClient class seems to be a general client instance, while this test needs a server instance. Think we can leave this  as is; the types don't do much.